### DISCUSSION TITLE
How can I execute/send a command from mpv-android to termux?

### DISCUSSION THREAD

**Discussion by erickyun** [termux/termux-app#3072] [Q&A] [answered]
Hi,

I so in mpv-android input.conf I wrote/tried
something like this but got no luck.
I would appreciate if someone could please help me?
 
```
0 run am start --user 0 -a com.termux.RUN_COMMAND -d mkdir /storage/emulated/0/download/IT-WORKED -n com.termux.app/TermuxActivity 
```

Note:
What I'm acutually trying to do is send a video link from mpv-android to yt-dlp so that it will download on android storage.

**Comment by agnostic-apollo** [comment, marked as answer]
Sending intent with wrong component and extras and the app hasn't requested the `RUN_COMMAND` permission anyways in its apk, so it won't work, ask the dev to add it.

https://github.com/termux/termux-app/wiki/RUN_COMMAND-Intent#top-command-with-am-startservice-command
